templates/journal.hbs

```diff
@@ -2,15 +2,19 @@
 {{#if this.hasContent}}
 <section class="category" data-category-id="{{ this.id }}" data-folder-id="{{ this.folder.id }}">
     <header class="category-header flexrow">
+        <i title="{{ localize 'WA.ButtonUnshrinkCategory' }}"
```

Comment:
Places like this are a good opportunity to use the new tooltips API instead of a standard HTML `title`
```suggestion
        <i data-tooltip="{{ localize 'WA.ButtonUnshrinkCategory' }}"
```

Reply:
I've put `data-tooltip=` where we had `title=`. I gives a nice display.

I'm curious:  Is there a way to alter those tooltip css ?
_I will surely use it in my custom system. But all my custom made tooltips are written in black, with a cornsilk background_